system/datatypes/festival.php

```diff
@@ -231,7 +231,7 @@ function showContributions() {
 		function getSettingsForm() {
 $html = <<<'EOF'
 <table>
-	<tr><td><label for="festival-title">Festival title</label> *</td><td><input name="festival-title"/></td></tr>
+	<tr><td><label for="festival-title">Festival title</label> *</td><td><input id="festival-title" name="festival-title"/></td></tr>
```

Comment:
Nitpick: This is a bit of a different change from the rest of the typo fixes, so this might be better placed into its own commit? Not essential, though.

Reply:
True, you are right. But let's not do it now, I'll commit more carefully next time.